etc/testing/kafka/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM golang:1.12.1
 RUN mkdir /app 
 ADD . /app/ 
-WORKDIR /app 
-ADD ./kafka-go /go/src/github.com/segmentio/kafka-go
+WORKDIR /app
+RUN go get github.com/segmentio/kafka-go
```

Comment:
Should we tie this to a release? (I *think* `go get <target>@<version>` works with vendoring on?)

Reply:
Actually, this didn't work due to the aforementioned semantic pathing issue. Even with go 1.13, `go get` using modules does not work in `$GOPATH`.